ui/reverse_proxy.go

```diff
@@ -77,16 +78,16 @@ func ReverseTequilapiProxy(tequilapiAddress string, tequilapiPort int, authentic
 			return
 		}
 
-		// authenticate all but the login route
-		if !isTequilapiURL(c.Request.URL.Path, endpoints.TequilapiLoginEndpointPath) {
-			cookieToken, err := c.Cookie(auth.JWTCookieName)
-
+		// authenticate all but the authentication routes
+		if !(isTequilapiURL(c.Request.URL.Path, endpoints.TequilapiAuthenticateEndpointPath) || isTequilapiURL(c.Request.URL.Path, endpoints.TequilapiLoginEndpointPath)) {
```

Comment:
nitpick: could we move this to a self contained func or restructure it some other way? It's kinda hard to read 🤷

Reply:
Improved